simple_history/models.py

```diff
@@ -311,3 +311,46 @@ def __get__(self, instance, owner):
         values = (getattr(instance, f.attname)
                   for f in self.model._meta.fields)
         return self.model(*values)
+
+
+class HistoricalChanges(models.Model):
+
+    class Meta:
+        abstract = True
+
+    def __sub__(self, old_history):
+        if not isinstance(old_history, type(self)):
+            raise TypeError(("unsupported operand type(s) for -: "
+                             "'{}' and '{}'").format(
+                            type(self),
+                            type(old_history)))
+        model_delta_class = getattr(self, 'model_delta_class', ModelDelta)
+        changes = {}
+        changed_fields = []
+        for field in self._meta.fields:
+            if hasattr(self.instance, field.name) and \
+               hasattr(old_history.instance, field.name):
+                old_value = getattr(old_history, field.name, '')
+                new_value = getattr(self, field.name)
+                if old_value != new_value:
+                    changes[field.name] = {'old_value': old_value,
+                                           'new_value': new_value}
+                    changed_fields.append(field.name)
+        return model_delta_class(changes,
+                                 changed_fields=changed_fields,
+                                 date=self.history_date,
+                                 user=self.history_user,
```

Comment:
I think maybe we should do neither here as well. If it becomes burdensome to type `my_delta.new_record.user` and `my_delta.old_record.user` we can make shortcuts later.

I'm open to re-considering and adding this and/or the historical date feature, but I'd like to not add them unless we're sure they're particularly helpful. One less feature to maintain? 😄